### ISSUE TITLE
Axis labels do not work in Mercator projection

### ISSUE THREAD

**Issue by MHBalsmeier** [SciTools/cartopy#1741]
In the Mercator projection axis labels are not drawn.

#### Code to reproduce

```
import cartopy.crs as ccrs;
import matplotlib.pyplot as plt;
from cartopy.mpl.gridliner import LONGITUDE_FORMATTER, LATITUDE_FORMATTER;

proj = ccrs.Mercator(central_longitude = 5);
ax = plt.axes(projection = proj);
ax.set_extent([0, 10, 0, 10], crs = ccrs.PlateCarree());
gl = ax.gridlines(crs=ccrs.PlateCarree(), draw_labels = True);
gl.xformatter = LONGITUDE_FORMATTER;
gl.yformatter = LATITUDE_FORMATTER;
plt.show();
```
#### Output

![test](https://user-images.githubusercontent.com/56491557/110036539-76877d00-7d3d-11eb-90a5-62d2e1d2e92f.png)

#### Details

Ubuntu20.04
cartopy0.18.0
matplotlib3.3.4

**Comment by MHBalsmeier** [comment]
Is this the same as issue #1731 maybe?
I am not using conda. If this is fixed in geos version 3.9.1, how can I install it on Ubuntu 20.04?
sudo apt-get install --upgrade libgeos-dev gives me 3.8.0-1build1.

**Comment by MHBalsmeier** [comment]
Reinstalled my system and now it works.
Weird, but seems to be rare, I am closing this.
